worlds/alttp/OverworldGlitchRules.py

```diff
@@ -220,26 +220,7 @@ def get_invalid_bunny_revival_dungeons():
     yield 'Sanctuary'
 
 
-def no_logic_rules(world, player):
-    """
-    Add OWG transitions to no logic player's world
-    """
-    create_no_logic_connections(player, world, get_boots_clip_exits_lw(world.mode[player] == 'inverted'))
```

Comment:
`create_no_logic_connections` is now unused as well, idk if you wanna remove that too